src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -339,41 +406,25 @@ Please enter the password to decrypt the content encryption key:
 
 **Note:** This section requires [`jq`](https://stedolan.github.io/jq/).
 
-1. Create a new JWK key pair:
+1. Create a new JWK key pair.
 
   ```shell
   step crypto jwk create jwk.pub jwk.priv
   ```
 
-2. Update the `"key"` field in the JWK provisioner configuration block.
-
-  Run the following, changing the provisioner name in the `jq` command to match your configuration:
-  
-  ```shell
-  jq --tab \
-     --argjson jwk "$(< jwk.pub )" \
-     '(.authority.provisioners[] | select(.name == "carl@example.com").key) = $jwk' \
-     $(step path)/config/ca.json > $(step path)/config/canew.json
-  ```
-
-  Inspect `$(step path)/config/canew.json` before replacing your `ca.json` with it.
+2. Update the provisioner.
 
-3. Format the private key and update the `"encryptedKey"` field in the provisioner configuration:
+  Run the following, changing the provisioner name in the command to match your configuration:
 
-  Run the following, changing the provisioner name in the `jq` command to match your configuration:
-  
   ```shell
-  ENCRYPTED_KEY=(step crypto jose format < jwk.priv)
-  jq --tab \
-     --arg encryptedkey "$ENCRYPTED_KEY" \
-     '(.authority.provisioners[] | select(.name == "carl@example.com".encryptedkey) = $encryptedkey' \
-      $(step path)/config/ca.json > $(step path)/config/canew.json 
+  step ca provisioner update you@smallstep.com \
```

Comment:
I believe you can also just do `step ca provisioner update you@smallstep.com --create`

Reply:
I've also added a `--no-private-key` flag to avoid storing the priv key.

Reply:
Or, remove it if was already stored.

Reply:
I'm considering removing `--no-private-key` and replacing with `--private-key ""` so don't quote me on this yet.

Reply:
`--no-private-key` was removed, and `--private-key ""` is the way to remove a private key